### PR TITLE
Update 37-ipfs-conf-dev-meetings-and-user-registry.md

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

**Comment by flyingzumwalt** [comment]
I lean towards also having an explicit Update at the top of the post, similar to the one we added at the top of the wikipedia post in https://github.com/ipfs/blog/commit/5074f78f6f60609a10ce0a70b367cdd2c6b71248

We could say something like

> _**Update** The target timeline for IPFS Conf has been changed to 2019. The original post listed a tentative date of November 2018._

File: content/post/37-ipfs-conf-dev-meetings-and-user-registry.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ title: "Announcements: IPFS Conf,  Developer Meetings for IPFS & libp2p and a Us
 author: flyingzumwalt
 ---
 
-We’re very excited to announce two events that have been in the making for a long time. In (tentatively) November we will host a public **IPFS Conf** in Lisbon. To prepare for that event, in July we will convene small **IPFS and libp2p Developer Meetings** in Berlin. All of these events will be loosely inspired by PyCon and Docker Conf — events where community members play a very active role in shaping the agenda, and where a large portion of time is allocated to working meetings where Working Groups and product teams make the most of being together in-person.
+We’re very excited to announce two events that have been in the making for a long time. In ~~(tentatively) November~~ 2019 we will host a public **IPFS Conf** in Lisbon. To prepare for that event, in July we will convene small **IPFS and libp2p Developer Meetings** in Berlin. All of these events will be loosely inspired by PyCon and Docker Conf — events where community members play a very active role in shaping the agenda, and where a large portion of time is allocated to working meetings where Working Groups and product teams make the most of being together in-person.
```
Comment (mikeal) [inline content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: We shouldn't commit to November, we're likely to do it earlier than that.

File: content/post/37-ipfs-conf-dev-meetings-and-user-registry.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ title: "Announcements: IPFS Conf,  Developer Meetings for IPFS & libp2p and a Us
...
 
 # IPFS Conf in November
 
-IPFS Conf will be a large, open-registration event for the extended ecosystem of people and projects that touch IPFS, libp2p, IPLD and multiformats.  We are aiming to host that event in Lisbon in November 2018. The event is open to all. To pre-register for IPFS Conf, use the [pre-registration form](https://goo.gl/forms/0Pu6VZzG8pRAmrrv2).
+IPFS Conf will be a large, open-registration event for the extended ecosystem of people and projects that touch IPFS, libp2p, IPLD and multiformats.  We are aiming to host that event in Lisbon in ~~November 2018~~ 2019. The event is open to all. To pre-register for IPFS Conf, use the [pre-registration form](https://goo.gl/forms/0Pu6VZzG8pRAmrrv2).
```
Comment (mikeal) [inline content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: This says 2018 *and* 2019 :)

Comment (flyingzumwalt) [reply content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: check out the rendered version -- he made them crossed out. 
> Lisbon in ~~November 2018~~ 2019

Comment (mikeal) [reply content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: huh, the diff view made this look so strange.

File: content/post/37-ipfs-conf-dev-meetings-and-user-registry.md (hunk 1 above)
Comment (diasdavid) [reply content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: My change is to strike through the ~~(tentatively) November~~ that was present before and add the 2019 not mentioning a month.

**Comment by diasdavid** [comment]
@flyingzumwalt good point. Added that sentence at the top.

**Comment by flyingzumwalt** [comment]
+1 Ill merge it. I also updated the Conf pre-registration form to say 2019 (the blog post links to it)

**Comment by flyingzumwalt** [comment]
@VictorBjelkholm  Is jenkins supposed to auto-deploy to the website when we merge PRs on this repo? If so, it's not working.

**Comment by VictorBjelkholm** [comment]
@flyingzumwalt last job on master was successful: https://ci.ipfs.team/job/IPFS/job/blog/job/master/28/console

And I can also see the updated wording over at https://blog.ipfs.io/37-ipfs-conf-dev-meetings-and-user-registry.md

Might been that you tried it out to early and you were still using a cache. More related to how DNS works rather than the auto-deploy itself.

**Pull request merged**